pom.xml

```diff
@@ -98,6 +98,8 @@
         <maven.site.plugin.version>3.9.1</maven.site.plugin.version>
         <nexus.staging.maven.plugin.version>1.6.8</nexus.staging.maven.plugin.version>
         <typesafe.config.version>1.4.2</typesafe.config.version>
+        <hudi.version>0.11.0</hudi.version>
```

Comment:
It seems that there are some dependency resolution issues with Spark 2.3 and 2.4. 
You can add the compatible version to the corresponding profile (line 400, tag <profiles>)